README.md

```diff
@@ -91,7 +103,7 @@ object GreetingClient extends ZIOAppDefault {
 
 ## Documentation
 
-Learn more on the [ZIO Http homepage](https://github.com/zio/zio-http)!
+Learn more on the [ZIO Http Docs](https://zio.dev/reference/)!
```

Comment:
This link is a reference to ZIO proper, not a reference to ZIO HTTP docs. I think a more specific link should be used.

Reply:
@jdegoes i changed it to https://zio.dev/zio-http/